### DISCUSSION TITLE
uploading LFS

### DISCUSSION THREAD

**Discussion by Srivastava-gdit** [git-lfs/git-lfs#5273] [General]
when pushing the LFS files it is showing this error

Git LFS is disable for this repository.

no finding in enterprise the option to enable.
![Screenshot 2023-01-31 155145](https://user-images.githubusercontent.com/105817792/215880363-e05cf096-6b6a-4bfd-839e-a6227fdf8e37.png)

**Comment by bk2204** [comment]
Hey,

When you get a response that says, "batch response", that means the server is giving you that response.  In this case, that server is GitHub.  You'll have to ask GitHub Support why it says Git LFS is disabled for your repository, since we won't know the answer.
